python/ray/experimental/workflow/api.py

```diff
@@ -60,7 +60,7 @@ def run(entry_workflow: "Workflow",
             type of the workflow will be ObjectRef[T] if the output step is
             a workflow step returning type T.
         storage: The external storage URL or a custom storage class. If not
-            specified, ``$(pwd)/workflow_data`` will be used.
+            specified, ``$(pwd)/.workflow_data`` will be used.
```

Comment:
I think a PR from Eric decides the directory should not be hidden (but in metaflow it is hidden). @ericl do you still consider the storage non-hidden?

Reply:
I'm thinking about moving the dir to `/tmp` ones due to runtime env.
With working dir in runtime env, it'll always upload the whole dir, which is not necessary for `workflow_data` dir.
Do you have some ideas there @ericl

Reply:
updates: synced with @ericl , I'm going to put it into a tmp dir and print warnings there to warn the user about tmp dir.